src/strongbox/specs.clj

```diff
@@ -198,31 +213,34 @@
 (s/def :addon/match (s/keys :req-un [::matched?]
                             :opt-un [::update?]))
 
-;; bare minimum required to find and 'expand' an addon
+;; bare minimum required to find and 'expand' an addon summary
 (s/def :addon/expandable
-  (s/keys :req-un [::name ::label
+  (s/keys :req-un [::name
+                   ::label
                    :addon/source ;; for host resolver dispatch
                    :addon/source-id ;; unique identifier for host resolver
                    ]
           :opt-un [::game-track-list ;; wowinterface only
                    ]))
 
-;; bare minimum required to install an addon
+;; the set of per-addon values provided by the remote host on each check
+(s/def :addon/source-updates
+  (s/keys :req-un [::version
+                   ::download-url
+                   ::game-track]
+          :opt-un [::interface-version]))
+
+;; result of expanding an addon
+(s/def :addon/expanded (s/merge :addon/expandable :addon/source-updates))
+
+;; bare minimum required to install an addon summary
 (s/def :addon/installable (s/merge
```

Comment:
quick grep for what is using `installable`, make sure they have test coverage

Reply:
surprisingly limited but there is a case where `expanded?` is being called that is essentially a check for `addon/installable`.